pwiz_tools/Skyline/Model/Prosit/Models/PrositModel.cs

```diff
@@ -432,8 +435,9 @@ public override int GetHashCode()
                     hashCode = (hashCode * 397) ^ (IntensityModel != null ? IntensityModel.GetHashCode() : 0);
                     hashCode = (hashCode * 397) ^ (RTModel != null ? RTModel.GetHashCode() : 0);
                     hashCode = (hashCode * 397) ^ (Settings != null ? Settings.GetHashCode() : 0);
-                    hashCode = (hashCode * 397) ^ (Precursor != null ? Precursor.GetHashCode() : 0);
                     hashCode = (hashCode * 397) ^ (Peptide != null ? Peptide.GetHashCode() : 0);
+                    hashCode = (hashCode * 397) ^ (Precursor != null ? Precursor.GetHashCode() : 0);

```

Comment:
In GetHashCode, we should use "RuntimeHelpers.GetHashCode" for fields that the Equals method uses ReferenceEquals to compare.
(Not something we should worry about in this pull request, and also has no actual impact other than being slightly less efficient)